diff --git a/node_modules/@analogjs/content/fesm2022/analogjs-content.mjs b/node_modules/@analogjs/content/fesm2022/analogjs-content.mjs
index 8ed62ee..eb5e285 100644
--- a/node_modules/@analogjs/content/fesm2022/analogjs-content.mjs
+++ b/node_modules/@analogjs/content/fesm2022/analogjs-content.mjs
@@ -1,5 +1,5 @@
 import * as i0 from '@angular/core';
-import { inject, Directive, HostListener, InjectionToken, ɵPendingTasksInternal as _PendingTasksInternal, Injectable, TransferState, makeStateKey, Component, ViewEncapsulation, Input, NgZone, PLATFORM_ID, ViewContainerRef, ViewChild } from '@angular/core';
+import { inject, Directive, HostListener, InjectionToken, ɵPendingTasks as _PendingTasksInternal, Injectable, TransferState, makeStateKey, Component, ViewEncapsulation, Input, NgZone, PLATFORM_ID, ViewContainerRef, ViewChild } from '@angular/core';
 import { DOCUMENT, Location, AsyncPipe, isPlatformBrowser } from '@angular/common';
 import { Router, ActivatedRoute } from '@angular/router';
 import { isObservable, firstValueFrom, of, Observable, from } from 'rxjs';
